mdpow/fep.py

```diff
@@ -648,7 +647,7 @@ def _setup(self, component, lmbda, foreign_lambdas=None, **kwargs):
                           free_energy='yes',
                           couple_moltype=self.molecule,
                           init_lambda=lmbda,
-                          fep_lambdas=feplambdas,
+                          #fep_lambdas=feplambdas,
```

Comment:
This should remain. I found that this was an issue when I used BAR with the wrong MDP file. For BAR you need 'bar_opls.mdp'.

I changed the default for the input file in 6ad86fc4d642965b9ca30cf6cf57a44340070fa6.